Example/Swift/ViewController.swift

```diff
@@ -132,9 +132,8 @@ class ViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthesizerD
              Otherwise, it will continue to compare the user to the old route and continually reroute the user.
              */
             self.navigation?.routeProgress = RouteProgress(route: self.userRoute!)
+            self.navigation?.routeProgress.currentLegProgress.stepIndex = 0
```

Comment:
@1ec5 because they are tracked independently. Noted that this should change in the PR notes above.